subsys/jwt/jwt.c

```diff
@@ -127,15 +127,16 @@ static int base64_append_bytes(const char *bytes, size_t len,
 	return 0;
 }
 
-struct jwt_header {
-	char *typ;
-	char *alg;
-};
-
-static struct json_obj_descr jwt_header_desc[] = {
-	JSON_OBJ_DESCR_PRIM(struct jwt_header, alg, JSON_TOK_STRING),
-	JSON_OBJ_DESCR_PRIM(struct jwt_header, typ, JSON_TOK_STRING),
-};
+static const char jwt_header[] =
+#ifdef CONFIG_JWT_SIGN_RSA
```

Comment:
Can you add a bit more to the comments about exactly how these values are created.